src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_Create.cs

```diff
@@ -142,7 +144,7 @@ internal static string GetDefaultTestSourceDocumentName(int index, string extens
             string workspaceKind = null,
             string extension = null,
             bool commonReferences = true,
-            bool openDocuments = true)
+            bool openDocuments = false)
```

Comment:
curious, is false better than true because we won't unnecessarily open docs / get text buffers when not needed?  

Would having the default be false create hard to figure out errors if someone is expecting to have the text buffers when the workspace is created (or is the error clear)

Reply:
So if somebody asks for a buffer we then open the document and produce a text buffer. The places where you still need true are when the side effect of opening a buffer is bad. Switching this broke maybe 40 or 50 tests in the entire codebase, so the default seems fine.

And yes the intent is unless you actually need text buffers, don't use them. :smile: